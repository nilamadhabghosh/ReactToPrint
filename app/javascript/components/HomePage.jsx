import React from 'react';
import Spacer from 'terra-spacer';
import { injectIntl, intlShape } from 'react-intl';

const HomePage = ({ intl }) => (
  <Spacer padding="medium">
    <h1>{intl.formatMessage({ id: 'HelloWorld' })}</h1>
  </Spacer>
);

HomePage.propTypes = {
  intl: intlShape
};

export default injectIntl(HomePage);
