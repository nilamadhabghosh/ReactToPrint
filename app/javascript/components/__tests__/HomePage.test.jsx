import React from 'react';
import { shallowWithIntl } from '@healtheintent/app-sdk';
import HomePage from '../HomePage';

describe('<HomePage />', () => {
  it('renders correctly', () => {
    const wrapper = shallowWithIntl(
      <HomePage.WrappedComponent />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
