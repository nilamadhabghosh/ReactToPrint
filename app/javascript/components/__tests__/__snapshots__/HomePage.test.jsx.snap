// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> renders correctly 1`] = `
<Spacer
  isInlineBlock={false}
  marginBottom="none"
  marginLeft="none"
  marginRight="none"
  marginTop="none"
  padding="medium"
  paddingBottom="none"
  paddingLeft="none"
  paddingRight="none"
  paddingTop="none"
>
  <h1>
    HelloWorld
  </h1>
</Spacer>
`;
